src/vm/checker/read_only/mod.rs

```diff
@@ -126,9 +141,23 @@ impl <'a, 'b> ReadOnlyChecker <'a, 'b> {
             Add | Subtract | Divide | Multiply | CmpGeq | CmpLeq | CmpLess | CmpGreater |
             Modulo | Power | BitwiseXOR | And | Or | Not | Hash160 | Sha256 | Keccak256 | Equals | If |
             ConsOkay | ConsError | DefaultTo | Expects | ExpectsErr | IsOkay | IsNone |
-            ListCons | GetBlockInfo | FetchEntry | FetchContractEntry | TupleGet | Print | AsContract | Begin => {
+            ListCons | GetBlockInfo | TupleGet | Print | AsContract | Begin => {
                 self.are_all_read_only(true, args)
             },
+            FetchEntry | FetchContractEntry => {
+                use vm::functions::tuples;
```

Comment:
Please put `use` statements at the top of the file.  Thanks!

Reply:
@jcnelson I think I've added this "local" `use` statements in a previous PR -- seemed like the consistent thing to use in relation to the surrounding code. Do we have a stylistic guide for stuff like this? Is this just stylistic?

Reply:
Generally speaking, `use` statements should come at the beginning of a file.  The only exceptions I can think of are:

* when there's an unavoidable namespace collision that would prevent this (e.g. `sha2::Digest` and `ripemd160::Digest` are one case)
* when you're using something conditionally (e.g. as part of `#[cfg(test)]` or the like).

Reply:
Good catch, thanks.

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1007/commits/2690b234d05d716c6679b7af84e3f38e80b60ebc